google-cloud-debugger/.repo-metadata.json

```diff
@@ -2,5 +2,6 @@
     "name": "clouddebugger",
     "language": "ruby",
     "distribution_name": "google-cloud-debugger",
-    "client_documentation": "https://googleapis.dev/ruby/google-cloud-debugger/latest"
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-debugger/latest",
+    "library_type": "GAPIC_MANUAL"
```

Comment:
Now that I look at these, I think this one is a weird case, and probably better labeled as INTEGRATION.